### PR TITLE
gdas_init utility bug fix

### LINKED ISSUE #908: gdas_init routine fails when processing GDAS v15 data
When coldstarting a global cycle with v15 GDAS or v16 retro GDAS data, the chgres step fails because the orography data for the input grid does not exist.  The orography data is required when using the 'warm' restart files as input to chgres because the orography record is not included in the restart files.

I think the problem started when the orography files were updated at #881.

### PR DESCRIPTION
**Pull request by GeorgeGayno-NOAA**
## DESCRIPTION OF CHANGES: 
When using warm restart files as input to the chgres step (v15/GDAS and v16 retro/GDAS options), it would fail because it was reading from the wrong input data orography directory. Update the scripts to point to the correct directory.

## TESTS CONDUCTED: 
Two test cases were done on Hera, Jet and WCOSS2. 

- Test 1
  - 2020/12/20/12z cycle
  - CDUMP=GDAS
  - use_v16retro=no

- Test 2
  - 2019/12/20/12
  - CDUMP=GDAS
  - use_v16retro=yes

Both tests completed normally and produce the required coldstart files. 

The 's4' scripts were tested by @DavidHuber-NOAA.

## DOCUMENTATION:
N/A

## ISSUE: 
Fixes #908.

### CODE REVIEW

**Comment by GeorgeGayno-NOAA** [comment]
@CoryMartin-NOAA tested the branch on the case that was failing for him. It worked.

**Review by KateFriedman-NOAA — APPROVED** [review]
Looks good! Thanks @GeorgeGayno-NOAA !

**Review by DavidHuber-NOAA — APPROVED** [review]
I gave this a try out on Jet and was able to create ICs.  Everything looks good.  Thanks!

**Comment by GeorgeGayno-NOAA** [comment]
> I gave this a try out on Jet and was able to create ICs. Everything looks good. Thanks!

Thanks, can you also try on 's4'?  I don't have access to that machine.

**Comment by DavidHuber-NOAA** [comment]
Sure, I'll let you know how that goes.

File: util/gdas_init/driver.s4.sh
Comment (DavidHuber-NOAA) [inline util/gdas_init/driver.s4.sh:1]: I did need to make one change on S4, after which I was able to run the utility to completion.  Line 18 should be changed to
```
module load prod_util/2.1.1
```

Comment (GeorgeGayno-NOAA) [reply util/gdas_init/driver.s4.sh:1]: Thanks! I will make the change.

Comment (GeorgeGayno-NOAA) [reply util/gdas_init/driver.s4.sh:1]: Done at 5a92fdb.

**Review by DavidHuber-NOAA — APPROVED** [review]
Looks good, thanks!

**Pull request merged**